.github/workflows/database.yml

```diff
@@ -0,0 +1,74 @@
+name: Database
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches:
+      - master
+      - 1.1.x
+
+env:
+  PYTEST_WORKERS: "auto"
+  SQL: 1
+
+jobs:
+  Linux_py38_slow:
+    name: Linux_py38_slow
+    runs-on: ubuntu-latest
+
+    env:
+      CONDA_PY: "38"
+      ENV_FILE: ci/deps/actions-38-slow.yaml
+      PATTERN: slow
+
+    services:
+      postgres:
+        image: postgres
+
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+
+        ports:
+        - 5432/tcp
+
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+
+    - name: Setting conda path
+      run: echo "${HOME}/miniconda3/bin" >> $GITHUB_PATH
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup MySQL
+      run: 
+        sudo systemctl enable mysql.service
+        sudo systemctl start mysql.service
+
+    - name: Setup environment and build pandas
+      run: ci/setup_env.sh
+      env:
+        POSTGRES_HOST: localhost
+        POSTGRES_PORT: ${{ job.services.postgres.ports[5432] }} 
+      if: always()
```

Comment:
if we're going to use githubactions, we may get better environment setup time using [conda incubator](https://github.com/conda-incubator/setup-miniconda)

e.g.

```
      - uses: actions/checkout@v2
      - name: Cache conda
        uses: actions/cache@v1
        env:
          # Increase this value to reset cache if environment-dev-py36.yml has not changed
          CACHE_NUMBER: 0
        with:
          path: ~/conda_pkgs_dir
          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
            hashFiles('environment.yml') }}
      - uses: conda-incubator/setup-miniconda@v2
        with:
          activate-environment: pandas-dev
          channel-priority: strict
          environment-file: environment.yml
          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
```